src/app/config.rs

```diff
@@ -86,11 +86,11 @@ impl Config {
     }
 
     pub fn build() -> Self {
-        let mut config = Self::default();
-
-        if let Some(config_from_file) = Self::detect_patch_hub_config_file() {
-            config = config_from_file;
-        }
+        let mut config = Self::detect_patch_hub_config_file().unwrap_or_else(|| {
+            let config = Self::default();
+            let _ = config.save_patch_hub_config();
```

Comment:
why are we ignoring possible errors from `save_patch_hub_config`?

Reply:
This is a temporary solution since this `save_patch_hub_config` method always fails in the test suit